### ISSUE TITLE
Video capture black and white

### ISSUE THREAD

**Issue by patriot1889** [hyperion-project/hyperion.ng#600]
If I start Hyperion.ng I get a a black and white image from the capture, all the LEDs are white.

If I start the "old" hyperion first, stop it, then start hyperion.ng everything is in colour.

> ~/hyperng/deploy $ ./hyperiond --debug
[hyperiond DAEMON] <DEBUG> <PythonInit.cpp:20:PythonInit()> Initializing Python interpreter
[hyperiond SettingsManager] <DEBUG> <SettingsManager.cpp:101:SettingsManager()> Settings database initialized
[hyperiond EFFECTFILES] <INFO> 36 effects loaded from directory :/effects/
[hyperiond EFFECTFILES] <INFO> 19 effect schemas loaded from directory :/effects/schema/
[hyperiond EFFECTFILES] <INFO> 0 effects loaded from directory /home/pi/.hyperion/custom-effects
[hyperiond SettingsManager] <DEBUG> <SettingsManager.cpp:101:SettingsManager()> Settings database initialized
[hyperiond BLACKBORDER] <DEBUG> <BlackBorderProcessor.cpp:65:handleSettingsUpdate()> Set mode to: default
[hyperiond ComponentRegister] <DEBUG> <ComponentRegister.cpp:67:componentStateChanged()> Blackborder detector: enabled
[hyperiond DAEMON] <INFO> set screen capture device to 'dispmanx'
[hyperiond DISPMANXGRABBER] <DEBUG> <Grabber.cpp:33:setVideoMode()> Set videomode to 0
[hyperiond DISPMANXGRABBER] <ERROR> Failed to open display! Probably no permissions to access the capture interface
[hyperiond DISPMANXGRABBER] <INFO> Capture interface is now disabled
[hyperiond DAEMON] <INFO> DISPMANX frame grabber created
[hyperiond V4L2:/dev/video0] <DEBUG> <Grabber.cpp:33:setVideoMode()> Set videomode to 0
[hyperiond V4L2:/dev/video0] <INFO> Signal threshold set to: {12, 12, 12}
[hyperiond V4L2:/dev/video0] <INFO> Cropping image: width=0 height=0; crop: left=12 right=17 top=3 bottom=3 
[hyperiond V4L2:/dev/video0] <INFO> Signal detection area set to: 0.250000,0.250000 x 0.750000,0.750000
[hyperiond DAEMON] <DEBUG> <hyperiond.cpp:420:handleSettingsUpdate()> V4L2 grabber created
[hyperiond JSONSERVER] <DEBUG> <JsonServer.cpp:25:JsonServer()> Created instance
[hyperiond JSONSERVER] <INFO> Started on port 19444
[hyperiond ComponentRegister] <DEBUG> <ComponentRegister.cpp:67:componentStateChanged()> LED device: enabled
[hyperiond LEDDEVICE] <INFO> LedDevice 'apa102' configured.
[hyperiond LEDDEVICE] <DEBUG> <ProviderSpi.cpp:48:open()> _baudRate_Hz 1000000,  _latchTime_ns 1
[hyperiond ComponentRegister] <DEBUG> <ComponentRegister.cpp:67:componentStateChanged()> Smoothing: enabled
[hyperiond LEDDEVICE] <DEBUG> <ProviderSpi.cpp:49:open()> _spiDataInvert 0,  _spiMode 0
[hyperiond FLATBUFSERVER] <INFO> Started on port 19400
[hyperiond FLATBUFCONNECTION] <INFO> Connecting to Hyperion: 127.0.0.1:19401
[hyperiond PROTOSERVER] <INFO> Started on port 19445
[hyperiond EFFECTENGINE] <INFO> run effect Rainbow swirl fast on channel 0
[hyperiond HYPERION] <DEBUG> <PriorityMuxer.cpp:153:registerInput()> Register new input 'System/EFFECT' with priority 0 as inactive
[hyperiond HYPERION] <INFO> Initial foreground effect 'Rainbow swirl fast' started
[hyperiond HYPERION] <DEBUG> <PriorityMuxer.cpp:153:registerInput()> Register new input 'System/V4L' with priority 240 as inactive
[hyperiond ComponentRegister] <DEBUG> <ComponentRegister.cpp:67:componentStateChanged()> V4L capture device: enabled
[hyperiond V4L2:/dev/video0] <INFO> configured v4l device: /dev/video0
[hyperiond BOBLIGHT] <DEBUG> <BoblightServer.cpp:28:BoblightServer()> Instance created
[hyperiond V4L2:/dev/video0] <DEBUG> <V4L2Grabber.cpp:600:init_device()> width=720 height=480
[hyperiond V4L2:/dev/video0] <DEBUG> <V4L2Grabber.cpp:646:init_device()> Pixel format=YUYV
[hyperiond WEBSERVER] <DEBUG> <WebServer.cpp:98:handleSettingsUpdate()> Set document root to: :/webconfig
[hyperiond WEBSERVER] <INFO> Started on port 8090 name 'Hyperion Webserver'
[hyperiond V4L2:/dev/video0] <INFO> Started
[hyperiond HYPERION] <INFO> Hyperion instance 'TV LED' has been started
[hyperiond HYPERION] <DEBUG> <PriorityMuxer.cpp:331:setCurrentTime()> Set visible priority to 0
[hyperiond HYPERION] <DEBUG> <PriorityMuxer.cpp:231:setInputImage()> Priority 240 is now active
[hyperiond HYPERION] <DEBUG> <PriorityMuxer.cpp:231:setInputImage()> Priority 0 is now active
[hyperiond HYPERION] <DEBUG> <PriorityMuxer.cpp:291:setCurrentTime()> Timeout clear for priority 0
[hyperiond HYPERION] <DEBUG> <PriorityMuxer.cpp:331:setCurrentTime()> Set visible priority to 240

<img width="526" alt="Screenshot 2019-08-14 at 17 22 07" src="https://user-images.githubusercontent.com/2946724/63042390-ce436680-bec1-11e9-9a3e-21cbc8c4949f.png">

Any thoughts on this?

**Comment by Paulchen-Panther** [comment]
This is similar to https://hyperion-project.org/threads/hyperion-ng-auf-raspberry-4-raspbian-buster-usb-grabber.3689/#post-15774
Will this weekend look what it is

**Comment by patriot1889** [comment]
Ah, yes! That looks like the same issue. I tried to search but couldn't find anything as it was actually in German!

Please let me know if there is anything you need me to test or log.

Thanks

**Comment by Paulchen-Panther** [comment]
PR is online #606

**Comment by Paulchen-Panther** [comment]
Can you please test the changes?

**Comment by patriot1889** [comment]
Sorry, I have only just had the chance to sit down and have a look. Unfortunately it doesn't seem like that has fixed the issue.

**Comment by patriot1889** [comment]
Currently, to negate this issue, I have modified my services to run
`v4l2-ctl --set-standard 0x000000ff`
before starting Hyperion.ng

I have also noticed that a black screen causes the LEDs to pulsate a dark blueish colour, which I have narrowed down to a brightness issue.

If I run a v4l2-ctl command to change the brightness it fixes it, even though v4l2-ctl comes back saying it is an invalid command.

Brightness reports as 448. If I run `v4l2-ctl --set-ctrl brightness=449` v4l2-ctl reports that the command is invalid, but it fixes the flickering LEDs. Afterwards, the brightness still reports 448.

Note that `v4l2-ctl --set-ctrl brightness=448` does not fix the flickering LEDs on black.


I thought I would mention these in case they are related to this issue.
